core/src/main/java/org/mule/runtime/core/internal/processor/strategy/StreamEmitterProcessingStrategyFactory.java

```diff
@@ -147,6 +147,7 @@ public void stop() {
       if (allSchedulersStopped()) {
         stopSchedulersIfNeeded();
       }
+      super.stop();
```

Comment:
should this be added to other implementations of PSs?

Reply:
AbstractReactorStreamProcessingStrategy is the one that introduces the inflightEvents concept and was not implementing Stoppable. I did not found other PS extending such class or incorporating a similar concept.